supply-chain/audits.toml

```diff
@@ -1,6 +1,17 @@
 
 # cargo-vet audits file
 
+[[audits.addr2line]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+version = "0.17.0"
+notes = "This version is used in rust's libstd, so effectively we're already trusting it"
+
+[[audits.addr2line]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.17.0 -> 0.19.0"
```

Comment:
Note that even upgrades such as this should have a `notes` listed for what was checked during the audit. Can you fill this in a bit? I looked at the diff myself and everything looks ok, but it would be good to mention here that dependency updates happened, some minor code movement, but nothing out of scope of what `addr2line` is supposed to do.